src/estimation/CMakeLists.txt

```diff
@@ -6,6 +6,8 @@
 # https://www.gnu.org/licenses/old-licenses/lgpl-2.1.html
 # at your option.
 
+find_package(iDynTree REQUIRED)
```

Comment:
This prevents the project to configure correctly, as you require iDynTree to be installed before you compile it. It probably works on your machine because you already have an iDynTree installed.

Reply:
Corrected in https://github.com/robotology/idyntree/pull/571/commits/16f4ad667ad39ce6594365c44d7164877360e7db

Thanks @prashanthr05